jbi/models.py

```diff
@@ -359,7 +360,25 @@ class ActionContext(Context, extra=Extra.forbid):
 
     rid: str
     operation: Operation
+    current_step: Optional[str]
     event: BugzillaWebhookEvent
     jira: JiraContext
     bug: BugzillaBug
     extra: dict[str, str] = {}
+    responses_by_step: dict[str, list] = {}
+
+    @property
+    def responses(self):
```

Comment:
Is there a reason to have both the `responses_by_step` property and this flattened `responses` representation? Is it was because we didn't want to make the scope of this change too large?

Reply:
Mainly because the API view returns a list of responses. We can change it and make return a map of responses, why not...